src/SFA.DAS.PensionsRegulator.Database/Tables/OrganisationAddress.sql

```diff
@@ -0,0 +1,21 @@
+﻿CREATE TABLE [dbo].[OrganisationAddress]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
+    [Employer_SK] INT NOT NULL, 
+    [OrganisationFullName] VARCHAR(255) NOT NULL, 
+    [OrganisationFullAddress] VARCHAR(255) NOT NULL, 
+    [AddressLine1] VARCHAR(35) NULL, 
+    [AddressLine2] VARCHAR(35) NULL, 
+    [AddressLine3] VARCHAR(35) NULL, 
+    [AddressLine4] VARCHAR(35) NULL, 
+    [AddressLine5] VARCHAR(35) NULL, 
```

Comment:
Should these be Line1 - Line5 to match the naming of the domain object? Or vice versa.

Reply:
Column names are according to design document.

"Addrress" seemed redundant for properties on an "Address" object. So kept them different and they are manually mapped anyway. Don't mind changing, particularly, but also don't see a problem with them being different :)